src/main/kotlin/io/github/mojira/arisa/modules/PiracyModule.kt

```diff
@@ -18,8 +18,8 @@ class PiracyModule(
                 piracySignatures,
                 "$description $environment $summary"
             ).bind()
-            addComment(CommentOptions(message)).toFailedModuleEither().bind()
-            resolveAsInvalid().toFailedModuleEither().bind()
+            addComment(CommentOptions(message))
```

Comment:
add comment should be always after resolving, and it should only happen if the module didnt fail to do the resolve